lib/css/modules/_showImages.scss

```diff
@@ -4,3 +4,49 @@ body:not(.res-showImages-displayImageCaptions) {
 		display: none;
 	}
 }
+
+.madeVisible {
```

Comment:
You probably want div.madeVisible or a.madeVisible depending on which one you're targeting.